shared/common-adapters/markdown.native.js

```diff
@@ -37,6 +37,7 @@ const codeSnippetBlockTextStyle = {
 
 const quoteBlockStyle = {borderLeftColor: globalColors.lightGrey2, borderLeftWidth: 3, paddingLeft: 8}
 
+// Setting undefined here overrides styles applied via Markdown
```

Comment:
Thanks for commenting this. A nitpick :the styles are applied by the `Text` component -- we want styles applied to the `Markdown` component to cascade and not be overridden by the `Text` component styles. Would you please clarify this?